GillianCore/GIL_Syntax/Constant.ml

```diff
@@ -5,10 +5,10 @@
 type t = TypeDef__.constant =
   | Min_float  (** The smallest float *)
   | Max_float  (** The largest float *)
-  | MaxSafeInteger  (** {% 2^53 - 1 %} **)
+  | MaxSafeInteger  (** [2^53 - 1] *)
   | Epsilon  (** Smallest positive number *)
   | Random  (** A random number between 0 and 1 *)
-  | Pi  (** The number {% \pi %} *)
+  | Pi  (** The number [pi] *)
```

Comment:
Wouldn't we rather have `{m \pi}` ?

Reply:
Applies to `{m 2^53 - 1}`

Reply:
I noticed the `{m }` syntax in the odoc docs about an hour after pushing this :P

Reply:
Wanna have fun? Look at who wrote the PR which introduces it :p